app/utils/PluginLoader.ts

```diff
@@ -24,6 +25,9 @@ export default class PluginLoader {
     function importAll(r: any, property: string) {
       Object.keys(r).forEach((key: string) => {
         const id = key.split("/")[3];
+        if (id === "github" && env.GITHUB_INTEGRATION_DISABLED) {
```

Comment:
Could just make this a generic `PLUGINS_DISABLED` – won't need to remove it later then?